minor doc improvements (#2006)
tox-dev/tox: docs/user_guide.rst

        # NOTE: you can run any command line tool here - not just tests
        pytest

> You can also try generating a ``tox.ini`` file automatically by running ``tox-quickstart`` and then answering a few

Author: ```suggestion
You can also try generating a ``tox.ini`` file automatically by running ``tox quickstart`` and then answering a few
```

   change.

   3. **commands**: run the specified commands in the specified order. Whenever the exit code of any of them is not

>    zero, stop and mark the environment failed. You can ignore the exit code when you start a command with a single dash
>    character.

Author: ```suggestion
   zero, stop and mark the environment failed. When you start a command with a dash character, the exit code will be ignored.
```
